### PR TITLE
Makes all pages share the same header, footer and classify device function

### PR DESCRIPTION
**Pull request by Taqdeer**
1) Refactors the header and footer of all views in a separate file
2) Refactors the code for classifyDevice in a separate file
3) Puts up the hiring status for developers as a comment in index.html

### CODE REVIEW

File: ~elm/src/HeaderFooter.elm [hunk 1]
```diff
@@ -0,0 +1,158 @@
+module HeaderFooter exposing (footer, header, phoneFooter, phoneHeader)
```
Comment (pre63) [inline ~elm/src/HeaderFooter.elm:1]: Inappropriate name.

File: ~elm/src/HeaderFooter.elm [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            { src = "/images/logo.svg"
+            , description = "Flint"
+            }
+        ]
+    , row [ centerX, width fill, Font.size 10 ]
+        [ el [ centerX ] <| text "© 2021 Flint, all rights reserved" ]
+    ]
+
+
+header : List (Element msg)
```
Comment (pre63) [inline ~elm/src/HeaderFooter.elm:93]: Why not take a device and return the correct view?

File: ~elm/src/Careers/View.elm [hunk 3]
```diff
@@ -105,13 +99,7 @@ phoneLayout model =
...
         , spacingXY 0 20
         ]
-        [ row []
-            [ Element.link
-                []
-                { url = toPath Home
-                , label = Element.image [ centerY, alignLeft, width (px 100), height (px 50) ] { src = "/images/logo.svg", description = "Flint" }
-                }
-            ]
+        [ row [ width fill ] phoneHeader
```
Comment (pre63) [inline ~elm/src/Careers/View.elm:102]: Why did we choose to not put the row (width fill) in the header component?

File: ~elm/src/ClassifyDevice.elm [hunk 4]
```diff
@@ -0,0 +1,26 @@
+module ClassifyDevice exposing (classifyDevice)
```
Comment (pre63) [inline ~elm/src/ClassifyDevice.elm:1]: module of the same name as a single function?

What is this module about? Mobile or responsive maybe?

Comment (Taqdeer) [reply ~elm/src/ClassifyDevice.elm]: Renamed.

File: ~elm/src/Careers/Update.elm [hunk 5]
```diff
@@ -2,10 +2,10 @@ module Careers.Update exposing (init, update)
 
 import Browser.Dom
 import Careers.Types exposing (Job, JobsStatus(..), Model, Msg(..))
-import Element exposing (Device, DeviceClass(..), Orientation(..))
 import Http
 import Maybe exposing (withDefault)
 import Regex
+import ResponsiveDesign exposing (classifyDevice)
```
Comment (pre63) [inline ~elm/src/Careers/Update.elm:8]:
```suggestion
import Device exposing (classify)
```

File: ~elm/src/Careers/View.elm [hunk 6]
```diff
@@ -37,8 +31,8 @@ import Element.Background as Background
 import Element.Border as Border
 import Element.Font as Font
 import List
-import Router.Routes exposing (Page(..), toPath)
-import Styles exposing (colors, heading, textStyles)
+import StandardPageLayout exposing (footer, header)
```
[the remark this answers is not in the document]
Comment (Taqdeer) [reply ~elm/src/Careers/View.elm]: Renamed.

**Pull request merged**